apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -163,14 +169,14 @@ export function ConversationView({
         featureType={FeatureType.Chat}
         isInvalid={isInvalid}
         containerClassName={classNames(
-          isSelectMode && !isExternal && 'group-hover/conversation-item:hidden',
+          isSelectMode && !isExternal && 'group-hover:hidden',
           isChosen && !isExternal && 'hidden',
         )}
       >
         {resourceToReview && !resourceToReview.reviewed && (
           <ReviewDot
             className={classNames(
-              'group-hover/conversation-item:bg-accent-secondary-alpha',
+              'group-hover:bg-accent-secondary-alpha',
```

Comment:
Are you sure that hover logic isn't broken?

Reply:
I checked and I don't think it's breaking anything.
BG of `<ShareIcon />` was broken because of the different groups.